src/rill.h

```diff
@@ -90,11 +90,15 @@ void rill_pairs_compact(struct rill_pairs *pairs);
 
 void rill_pairs_print(const struct rill_pairs *pairs);
 
+void rill_pairs_flip_values(struct rill_pairs* pairs);
```

Comment:
I would call this `rill_pairs_invert` but I tend to prefer overly concise and dense names :)

Reply:
Fixorzed